src/client-functions/client-function-builder.js

```diff
@@ -12,7 +12,7 @@ import getCallsite from '../errors/get-callsite';
 const DEFAULT_EXECUTION_CALLSITE_NAME = '__$$clientFunction$$';
 
 export default class ClientFunctionBuilder {
-    constructor (fn, options, callsiteNames = {}) {
+    constructor (fn, options, callsiteNames = {}, clientFunctionWrongTypeMessage = MESSAGE.clientFunctionCodeIsNotAFunction) {
```

Comment:
note that
`a regular or client function`
or
`a regular function or ClientFunction`
but not
~~`a regular or ClientFunction`~~
cause the latter does not contain the word `function` to which `regular` could relate